crypto/rand/rand_lib.c

```diff
@@ -241,10 +253,15 @@ size_t rand_drbg_get_additional_data(unsigned char **pout, size_t max_len)
 #endif
 
 #ifdef OPENSSL_SYS_UNIX
+# if HAVE_CLOCK_GETTIME
```

Comment:
I think you can remove the `OPENSSL_SYS_UNIX condition at this point, since it's a condition to have `HAVE_CLOCK_GETTIME` be 1

Reply:
But there is an `#elif defined(OPENSSL_SYS_WIN32)` which belongs to the `#ifdef OPENSSL_SYS_UNIX`.

Reply:
Even if I had omitted the `OPENSSL_SYS_UNIX` check, I would have had to add an `defined(HAVE_CLOCK_GETTIME)` check instead to avoid `-Wundef` warnings. So I didn't see any gain in omitting it.